core/modules/catalog/helpers/index.ts

```diff
@@ -584,3 +584,15 @@ export function configurableChildrenImages (product) {
   }
   return configurableChildrenImages
 }
+
+/**
+ * check if one of the configuableChildren has an image
+ */
+
+export function childHasImage (children) {
+  let hasImage = false;
+  children.forEach((child) => {
+    if (child.image.length == 0 || child.image === 'no_selection') hasImage = true
```

Comment:
I'm not sure I get this right - could you explain to me why a child has an image if its images have no length or if the image is `no_selection`? Shouldn't that be the opposite conditions?
also please concider to use `includes` instead of `forEach` and provide a default value for children, because with the wrong argument we could have an error here. If you'd add unit tests for this helper method I'd be really grateful :)